### PR TITLE
Fixed error finding directory

### PR DESCRIPTION
**Pull request by bjcmit**
Line 41 of gen_manifests.sh was throwing the following error: gen_manifests/arc-cicd-cluster/manifests/dev/azure-vote.yaml: No such file or directory.

### CODE REVIEW

File: utils/generate-manifests.sh [hunk 1]
```diff
@@ -7,8 +7,8 @@
 # 
 
 # For the Azure-Vote sample the following env varuables are supposed to be initialized: 
-# export VOTE_APP_TITLE='Awesome Voting App'
-# export AZURE_VOTE_IMAGE_REPO=gitopsflowacr.azurecr.io/azvote
+export VOTE_APP_TITLE='Awesome Voting App'
+export AZURE_VOTE_IMAGE_REPO=ahcontreg.azurecr.io/azvote
```
Comment (tcare) [inline utils/generate-manifests.sh:11]: Why the change?

File: utils/generate-manifests.sh [hunk 2]
```diff
@@ -7,8 +7,8 @@
 # 
 
 # For the Azure-Vote sample the following env varuables are supposed to be initialized: 
-# export VOTE_APP_TITLE='Awesome Voting App'
-# export AZURE_VOTE_IMAGE_REPO=gitopsflowacr.azurecr.io/azvote
+export VOTE_APP_TITLE='Awesome Voting App'
```
Comment (tcare) [inline utils/generate-manifests.sh:10]: We're setting this as part of the guide, shouldn't we leave these commented?

Comment (bjcmit) [reply utils/generate-manifests.sh:10]: We should leave these commented.

File: utils/generate-manifests.sh (hunk 1 above)
Comment (bjcmit) [reply utils/generate-manifests.sh:11]: The change is not needed.

**Review by tcare — APPROVED** [review]

**Comment by tcare** [comment]
LGTM, but let's clear the docs PR before we merge both.
